app/views/Cell.coffee

```diff
@@ -1,33 +1,38 @@
+$      = require 'jquery'
+Colors = require './colors'
+
 class Cell
 
-  constructor: (@col, @row, @size, @two, @board) ->
+  constructor: (@col, @row, @size, @two, @board, @clickHandler) ->
     @isDeleted = false
+    @isSelected = false
     @rect = @two.makeRectangle @getX(), @getY(), @size, @size
     @two.update()
 
-  setColor: (c) =>
+  setColor: (c) ->
     @color = c
     @rect.fill = c
     @two.update()
 
-  setBorder: (c) =>
+  setBorder: (c) ->
     @rect.stroke = c
     @rect.linewidth = 6
     @two.update()
 
-  hide: =>
-    @rect.opacity = 0
+  hide: ->
+    @rect.visible = false
     @two.update()
 
-  getX: (col = @col) =>
+  getX: (col = @col) ->
```

Comment:
because this allows for two calling modes
  1. `getX()` returns the current x-coord
  2. `getX col` returns the x-coord of `col`